README.md

```diff
@@ -42,18 +54,37 @@ Now that you know what layers you have, decide what data structure would be most
 After this point, consult with a neighbor.
 
 ### Step 3: Make the data structure!
-Create the data structure by starting with the outline, then manually input all the data from `rides.csv` into it.
+Create the data structure by starting with the outline. Then, assign this data structure to a variable. Give the variable a good name that represents what the outermost layer is.
+
+At this point, my code (for an unrelated data set) may look like this:
+
+```rb
+locations = [
```

Comment:
I see and like what you're doing here, trying to make sure they get it all in one variable. However, I have a couple concerns:

- The unrelated example may be confusing
- Showing this as an array of hashes may be too much of a hint

Given that you've stressed a couple other places that at the top level it should all fit in one variable, it might be you don't need this example.